actioncable/test/test_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'mocha/setup'
 
 require 'rack/mock'
+require 'active_support/core_ext/hash/indifferent_access'
```

Comment:
Why are we requiring this in the test helper? If we need it in `actioncable/lib/action_cable/connection/subscriptions.rb`, let's require it there.